test/ln.js

```diff
@@ -105,6 +105,24 @@ shell.ln('-s', 'ln', 'tmp/dir1');
 assert(fs.existsSync('tmp/ln/hello'));
 assert(fs.existsSync('tmp/dir1/hello'));
 
+// To current directory
+shell.cd('tmp');
+shell.ln('-s', './', 'dest');
+shell.touch('testfile.txt');
+assert(fs.existsSync('testfile.txt'));
+assert(fs.existsSync('dest/testfile.txt'));
+shell.rm('-f', 'dest');
+shell.mkdir('dir1');
+shell.cd('dir1');
+shell.ln('-s', './', '../dest');
+shell.touch('insideDir.txt');
+shell.cd('..');
+assert(fs.existsSync('testfile.txt'));
+assert(fs.existsSync('dest/testfile.txt'));
+assert(fs.existsSync('dir1/insideDir.txt'));
+assert(!fs.existsSync('dest/insideDir.txt'));
```

Comment:
I think it shouldn't. Symbolic relative links are relative to the destination, hard links are relative to the cwd.

`insideDir.txt` is inside `dir1`, not its parent directory, and `dest` points at `dir1`'s parent

Try out this behavior with UNIX Ln and I think it should behave the same